pinc/misc.inc

```diff
@@ -13,6 +13,25 @@ function array_get($arr, $key, $default)
     }
 }
 
+function get_changed_fields_for_objects($new, $old)
```

Comment:
this feels ideal for a unit test if you are bored :D

Reply:
Added some unit tests for this and `array_get()` while I was here!